msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -1276,7 +1277,7 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
 
         val result = application.signIn(emptyString, password)
         assertTrue(result is SignInError)
-        assertTrue((result as SignInError).errorType == null)
```

Comment:
I don't understand how this test passed in the past. A ClientException should have been thrown, but that's not in the @Test annotation 🤔

Reply:
note: the `@throws` can be removed from the comments

Reply:
Done

Reply:
Ah, when using Java SDK methods the SDK wraps exceptions that are thrown into `callback.onError(e)`.